frontend/src/views/Quiz.vue

```diff
@@ -55,13 +55,16 @@ function goToPreviousQuestion() {
 <template>
     <div class="quiz-container">
         <h1 class="quiz-title">
-            <span class="back-to-quiz-preview-icon">
+            <span
+                v-if="currentQuestionIndex != null"
```

Comment:
Have you replaced the `v-show` directive by `v-if` for any particular reason ?

Reply:
Yes. When you use `v-show`, the component is still there and occupy the place it should. While using `v-if` doesn't place the component at all in the DOM if the condition is not respected.

Reply:
Yes, that was the behavior I wanted. 
But with your new UI, I seen that  it's not useful anymore.